### PR TITLE
style(autoSuggest): update the style for auto suggest component

### PR DESCRIPTION
**Pull request by manthachan**
Update the auto suggest style
Changed the form fields font & height

screen shot 2018-01-31 at 3 08 51 pm

### CODE REVIEW

File: theme/layout/forms.less [hunk 1]
```diff
@@ -1,3 +1,3 @@
-@field-gutter-width: 15px;
+@field-gutter-width: 7px;
```
Comment (mengtzu) [inline theme/layout/forms.less]: 7px isn't on a 4px grid, is this to do with the border size?

File: react/Autosuggest/Autosuggest.js [hunk 2]
```diff
@@ -47,7 +48,8 @@ export default class Autosuggest extends Component {
     className: '',
     label: '',
     labelProps: {},
-    showMobileBackdrop: false
+    showMobileBackdrop: false,
```
Comment (mengtzu) [inline react/Autosuggest/Autosuggest.js]: you've named this `mobile` but it doesn't end up matching up to a media query, is there a more generic name for this behaviour?

Comment (manthachan) [reply react/Autosuggest/Autosuggest.js]: ok

File: theme/layout/forms.less (hunk 1 above)
Comment (manthachan) [reply theme/layout/forms.less]: should be, the form field element have 1px border

Comment (mengtzu) [reply theme/layout/forms.less]: Actually I see the one you changed didn't fit the grid either, same deal I guess 😹

Comment (hscgavin) [reply theme/layout/forms.less]: how about 8px?

**Pull request merged**